manim/mobject/geometry.py

```diff
@@ -1428,143 +1432,313 @@ def __init__(self, start_anchor, start_handle, end_handle, end_anchor, **kwargs)
-        :save_last_frame:
+        import numpy as np
 
-        class PolygonExample(Scene):
+        class PolygramExample(Scene):
             def construct(self):
-                isosceles = Polygon([-5, 1.5, 0], [-2, 1.5, 0], [-3.5, -2, 0])
-                position_list = [
-                    [4, 1, 0],  # middle right
-                    [4, -2.5, 0],  # bottom right
-                    [0, -2.5, 0],  # bottom left
-                    [0, 3, 0],  # top left
-                    [2, 1, 0],  # middle
-                    [4, 3, 0],  # top right
-                ]
-                square_and_triangles = Polygon(*position_list, color=PURPLE_B)
-                self.add(isosceles, square_and_triangles)
+                hexagram = Polygram(
+                    [[0, 2, 0], [-np.sqrt(3), -1, 0], [np.sqrt(3), -1, 0]],
+                    [[-np.sqrt(3), 1, 0], [0, -2, 0], [np.sqrt(3), 1, 0]],
+                )
+                self.add(hexagram)
+
+                self.play(MoveAlongPath(Dot(), hexagram), run_time=5, rate_func=linear)
+
+
     """
 
-    def __init__(self, *vertices, color=BLUE, **kwargs):
-        VMobject.__init__(self, color=color, **kwargs)
-        # There are actually four corners, and the first one is repeated twice to form the four vertices.
-        self.set_points_as_corners([*vertices, vertices[0]])
+    def __init__(self, *vertex_groups: Sequence[Sequence[float]], color=BLUE, **kwargs):
+        super().__init__(color=color, **kwargs)
+
+        for vertices in np.array(vertex_groups):
+            self.start_new_path(vertices[0])
+            self.add_points_as_corners([*vertices[1:], vertices[0]])
 
-    def get_vertices(self):
-        """Gets the vertices of the polygon.
+    def get_vertices(self) -> np.ndarray:
+        """Gets the vertices of the :class:`Polygram`.
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+            The vertices of the :class:`Polygram`.
 
         Examples
         --------
         ::
 
             >>> sq = Square()
-            >>> points = sq.get_vertices()
-            >>> points
+            >>> sq.get_vertices()
             array([[ 1.,  1.,  0.],
                    [-1.,  1.,  0.],
                    [-1., -1.,  0.],
                    [ 1., -1.,  0.]])
+        """
+
+        return self.get_start_anchors()
+
+    def get_vertex_groups(self) -> np.ndarray:
+        """Gets the vertex groups of the :class:`Polygram`.
 
         Returns
         -------
         :class:`numpy.ndarray`
-            Returns a list of the coordinates of polygon's vertices.
+            The vertex groups of the :class:`Polygram`.
+
+        Examples
+        --------
+        ::
+
+            >>> poly = Polygram([ORIGIN, RIGHT, UP], [LEFT, LEFT + UP, 2 * LEFT])
+            >>> poly.get_vertex_groups()
+            array([[[ 0.,  0.,  0.],
+                    [ 1.,  0.,  0.],
+                    [ 0.,  1.,  0.]],
+            <BLANKLINE>
```

Comment:
This seems a bit odd to me, but I guess it is required?

Reply:
It is required by doctest and just gets rendered as a normal blank line in the docs